tls_examples/AGS_AeroDB/aerospike.conf

```diff
@@ -0,0 +1,50 @@
+# Aerospike database configuration file.
```

Comment:
I think we should add some comments on the relevant stuff in this file. Lets talk in a bit.